flake.nix

```diff
@@ -164,6 +167,19 @@
         buildInputs = [ ] ++ extraBuildInputs;
       };
 
+      # Combine the environmnet with cargo args needed to build wasm package
+      noirWasmArgs = wasmEnvironment // {
+        pname = "noir_wasm";
+
+        src = ./.;
```

Comment:
Without using `cleanCargoSource`, I believe this will cause an entire rebuild on each change (likely deps and all)

Reply:
I think the `cleanCargoSource` is causing the shell scripts to not be available again. I'd be fine not having the shell scripts and relying on the nix builds so we don't have to rebuild everything from scratch each time. WDYT?

Reply:
deps won't rebuild, we're better with external scripts - 
1. not an unusual nix pattern
2. gives possibility to develop/build w/o nix

Reply:
@kobyhallx we would have to delete the .sh scripts and go back to the point before your PR was merged for this.